Audible-Altimeter/src/runner.cpp

```diff
@@ -65,7 +84,7 @@ bool Runner::read_event() {
   const auto current_altitude{data->altitude};
   if (m_state == State::IDLE && current_altitude > 1'000) {
     enter_state(State::ACTIVE);
-    m_audio_player.play(k_altitudes[m_index].sample_id);
+    m_audio_player.play(AUDIO_SAMPLE_ID::TEN);
```

Comment:
good thinking. I can make one of those in ableton and downsample it so that we can use it with your cmake setup. https://github.com/khurd21/Audible-Altimeter/issues/17